.appveyor.yml

```diff
@@ -1,13 +1,31 @@
 environment:
   matrix:
-  - nodejs_version: "0.12"
-  - nodejs_version: "4"
-  - nodejs_version: "6"
+  - nodejs_version: 0.12
+  - nodejs_version: 4
+  - nodejs_version: 6
 
-  GCLOUD_TESTS_KEY: 'key.appveyor.json'
+  GCLOUD_TESTS_KEY: key.appveyor.json
   GCLOUD_TESTS_KEY_SECRET:
     secure: usGkvXjqCzDNIQHx8oEncrZOmez7GBocB1EXM+g/9rw=
 
+cache:
+  - '%APPDATA%\npm-cache'
+  - node_modules -> package.json
+  - packages\bigquery\node_modules -> packages\bigquery\package.json
+  - packages\bigtable\node_modules -> packages\bigtable\package.json
+  - packages\compute\node_modules -> packages\compute\package.json
+  - packages\datastore\node_modules -> packages\datastore\package.json
+  - packages\dns\node_modules -> packages\dns\package.json
+  - packages\google-cloud\node_modules -> packages\google-cloud\package.json
+  - packages\language\node_modules -> packages\language\package.json
+  - packages\logging\node_modules -> packages\logging\package.json
+  - packages\prediction\node_modules -> packages\prediction\package.json
+  - packages\pubsub\node_modules -> packages\pubsub\package.json
+  - packages\resource\node_modules -> packages\resource\package.json
+  - packages\storage\node_modules -> packages\storage\package.json
+  - packages\translate\node_modules -> packages\translate\package.json
+  - packages\vision\node_modules -> packages\vision\package.json
```

Comment:
Oh, gotcha. With Travis, it separates caches by the Node version and/or (?) OS:

> There is one cache per branch and language version/ compiler version/ JDK version/ Gemfile location/ etc.
> \- https://docs.travis-ci.com/user/caching/#Fetching-and-storing-caches

With AppVeyor, it's not as clear from their docs. The AppVeyor builds with caching from this PR have been succeeding, so I'd say we're safe.